app/src/main/java/de/danoeh/antennapod/adapter/NavListAdapter.java

```diff
@@ -295,10 +296,15 @@ private View getSectionDividerView(View convertView, ViewGroup parent) {
         LayoutInflater inflater = (LayoutInflater) context
                 .getSystemService(Context.LAYOUT_INFLATER_SERVICE);
 
-        convertView = inflater.inflate(R.layout.nav_section_item, parent, false);
-
-        convertView.setEnabled(false);
-        convertView.setOnClickListener(null);
+        if (UserPreferences.getFeedFilter() != UserPreferences.FEED_FILTER_NONE) {
+            convertView = inflater.inflate(R.layout.nav_section_filter_divider, parent, false);
+            convertView.setEnabled(true);
+            convertView.setOnClickListener((l) -> FeedFilterDialog.showDialog(context));
+        } else {
+            convertView = inflater.inflate(R.layout.nav_section_item, parent, false);
+            convertView.setEnabled(false);
+            convertView.setOnClickListener(null);
```

Comment:
Could you please move click handling to `NavDrawerFragment.onItemClick`? All other items of the sidebar are handled in that file. It's also a good practice to keep displaying (adapter) and behavior (click listener) separated. I must admit that this is not always the case in the code base but in this case, it should be pretty easy to do.

Reply:
I tried this in 0a384bc but the on click event is not fired

Reply:
Setting the listener to null is not required here. I think this interferes more with the list than what it helps.